frontend/app/utils/zod-utils.ts

```diff
@@ -34,3 +36,11 @@ export function transformFlattenedError<T extends { fieldErrors: Record<string,
   const transformedEntries = Object.entries(flattenedError.fieldErrors).map(([key, value]) => [key, value.at(0)]);
   return Object.fromEntries(transformedEntries);
 }
+
+type FlattenedZodError = typeToFlattenedError<ZodError<string>>;
+
+export function getErrorList(flattenedError: FlattenedZodError) {
```

Comment:
```suggestion
export function getErrorList(flattenedError: FlattenedZodError): string[] {
```

Although this might be moot since we will be moving all validations to valibot (see #119).